src/helpers/soldProducts.js

```diff
@@ -0,0 +1,14 @@
+import {inventory} from '../constants/inventory.js';
+
+
+function soldProducts () {
```

Comment:
hier mag de spatie tussen soldProducts en () weggelaten worden! Dit is niet noodzakelijk voor de werking van de code maar wel netjes.